image-fetcher/fetch-image.sh

```diff
@@ -1,15 +1,23 @@
 #!/bin/sh
 set -ex
 
-gcloud auth activate-service-account \
+gcloud -q auth activate-service-account \
   --key-file=/secrets/gcr-pull.json
 
-gcloud auth configure-docker
+gcloud -q auth configure-docker
 
-while true
-do
-    for image in gcr.io/$PROJECT/base:latest $(curl -sSL http://notebook/worker-image) $(curl -sSL http://notebook2/worker-image) google/cloud-sdk:237.0.0-alpine
-    do
+DEFAULT_NAMESPACE=$(jq -r '.default_namespace' < /deploy-config/deploy-config.json)
+case $DEFAULT_NAMESPACE in
+    default)
```

Comment:
what is default here? do we need quotes?

Reply:
default is the name of the production namespace.  We don't need quotes.  * is the default case for shell case arms.